### ISSUE TITLE
Comcast Telecom Consumer Complaints.

### ISSUE THREAD

**Issue by Sanchu457** [Sanchu457/sanchit_portfolio#2]
Data Scientist Projects

Project 1 Mercedes-Benz Greener Manufacturing

Mercedes-Benz world’s biggest manufacturers of premium cars, the company’s engineers have developed a robust testing system ,optimizing the speed of their testing system for many possible feature combinations is complex and time-consuming without a powerful algorithmic approach,permutations of features in a Mercedes-Benz car to predict the time it takes to pass testing.Optimal algorithms will contribute to faster testing, resulting in lower carbon dioxide emissions without reducing Mercedes-Benz’s standards.
 
 I reduce the time that cars spend on the test bench
 
 If for any column(s), the variance is equal to zero , i remove those variable(s).
 
 Checked  null and unique values for test and train sets.
 
 Applied the  label encoder with help of Sklearn library.
 
 Performed dimensionality reduction.
 
 Predicted  test_df values using XGBoost with help of XFboost library.![2022-03-01 22](https://user-images.githubusercontent.com/95237822/156052348-6cc68474-57be-4113-9e7c-fa4184060269.jpg)

This is a pictorial view for comparison between the target for training data-set and predicted target for testing 
_
